src/frontend/web_application/src/components/ImportContactForm/presenter.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { v1 as uuidV1 } from 'uuid';
+import { FieldErrors } from '../form';
+import Button from '../Button';
+import Icon from '../Icon';
+
+import './style.scss';
+
+class ImportContactForm extends Component {
+  static propTypes = {
+    uploadFile: PropTypes.func,
+    onCancel: PropTypes.func,
+    __: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    uploadFile: null,
+    onCancel: null,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      files: [],
+      errors: [],
+    };
+
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onResetForm = this.onResetForm.bind(this);
+    this.renderButtons = this.renderButtons.bind(this);
+  }
+
+  onInputChange(ev) {
+    const file = ev.target.files[0];
```

Comment:
there's a pbm here I think:

In case the input change, it will try to validate `ev.target.files[0]` which might be undefined.

you should pass all files to validation method and expect to have at least 1 file  
or  
add a condition to have at least one file before going to validation, so it is a no-operation if input is empty